source/_components/device_tracker.mqtt.markdown

```diff
@@ -22,17 +22,36 @@ device_tracker:
     devices:
       paulus_oneplus: 'location/paulus'
       annetherese_n4: 'location/annetherese'
+    payload_home: present
```

Comment:
In the first example we want only required config variables, but feel free to add an full configuration example at the bottom 😉

Reply:
done, thanks!